PowerShell Scanners/Get Available Windows Updates/README.md

```diff
@@ -5,5 +5,7 @@
 
 Installs the PSWindowsUpdate module, then runs Get-WindowsUpdate. This returns all updates that are currently available.
 
+WSUS switch added to work with all networks
```

Comment:
Please use a `# Parameters` block for this. https://github.com/pdq/PowerShell-Scanners/blob/master/PowerShell%20Scanners/_Template/README.md

Reply:
Thanks, Colby. I got that added.